.github/lock.yml

```diff
@@ -8,7 +8,7 @@ daysUntilLock: 60
 skipCreatedBefore: false
 
 # Issues and pull requests with these labels will be ignored. Set to `[]` to disable
-exemptLabels: ['blocking', 'bug', 'effectiveness', 'efficiency', 'feature request', 'help wanted', 'in progress']
+exemptLabels: ['effectiveness', 'efficiency', 'feature request', 'help wanted']
 
 # Label to add before locking, such as `outdated`. Set to `false` to disable
 lockLabel: true
```

Comment:
maybe this should set to `false`?

Reply:
As we do not have `outdated` label or any similar to this, I think you are right.